app/src/main/res/drawable/ic_avatar_authenticated_no_picture.xml

```diff
@@ -6,6 +6,5 @@
   <path
       android:pathData="m12,22c-5.5228,0 -10,-4.4772 -10,-10s4.4772,-10 10,-10 10,4.4772 10,10c-0.0061,5.5203 -4.4797,9.9939 -10,10zM17.6655,14.2092c-1.4812,-1.7094 -3.5322,0.5908 -5.6658,0.5908 -2.1322,0 -4.1846,-2.3002 -5.6658,-0.5908 -0.42,0.4844 -0.4536,1.19 -0.0938,1.7192 1.2558,1.848 3.3558,3.0716 5.7596,3.0716 2.4038,0 4.5038,-1.2236 5.7596,-3.0716 0.3612,-0.5292 0.3262,-1.2348 -0.0938,-1.7192zM12,13.5c2.3474,0 4.25,-1.9026 4.25,-4.25 0,-2.3474 -1.9026,-4.25 -4.25,-4.25 -2.3474,0 -4.25,1.9026 -4.25,4.25 0,2.3474 1.9026,4.25 4.25,4.25z"
       android:fillColor="#f9f9fa"
-      android:fillAlpha=".8"
```

Comment:
Could you run the new colors by @brampitoyo to make sure they're what he intended?  Also, are there any other SVGs we should be updating this way?

Reply:
I'm going to remove this change and file a follow-up.